src/osemgrep/core/Metrics_.mli

```diff
@@ -1,3 +1,5 @@
+module Out = Semgrep_output_v1_t
```

Comment:
we avoid module aliases in .mli.
Just replace Out with the full name in the type signature.